server/webapp/WEB-INF/rails/webpack/helpers/ajax_poller.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as CONSTANTS from "helpers/constants";
+import * as _ from "lodash";
+import * as m from "mithril";
+import {Stream} from "mithril/stream";
+import * as stream from "mithril/stream";
+
+export interface Options<T> {
+  intervalSeconds: number;
+  inSeconds: number;
```

Comment:
Is this the initial interval? Can we rename it to `initialIntervalSeconds` to make it clear?